client/stylesheets/_toCat.scss

```diff
@@ -4616,21 +4623,23 @@ On click the images will be displayed at normal size to complete the effect
 
 .searchResultsCards {
     vertical-align: top !important;
-    margin-top: -965px !important;
+    margin-top: -1300px !important;
     float: right;
 }
 
 .searchResultsCardUl > li {
     display: inline-block;
     width: 85%;
-    padding: 10px;
-    border: 2px solid #00bfff;
+    padding: 13px 10px 10px 10px;
+    border: 1px solid #00bfff;
```

Comment:
Color literals like `#00bfff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces